views/results.ejs

```diff
@@ -1,2 +1,5 @@
 <% // Render your search results here %>
 <% // This is how to do comments in EJS, by the way... %>
+<a href="/movies/<%= t.imdbID %>" ><%= t.Title%></a>
+<h2> (<%= t.Year %>)</h2>
+<img src="<%= t.Poster %> " alt="Poster"> 
```

Comment:
You need to be iterating over the full set of data that this page has access to.  It could look something like this:

`
<% data.forEach(t=> { %>
  <a href="/movies/<%= t.imdbID %>"><%= t.Title %></a>
  <h2> (<%= t.Year %>)</h2>
  <img src="<%= t.Poster %>" alt="Poster">
<% }) %>
`